### PR TITLE
added hover tool, sd selector, log normal masses prior

### PR DESCRIPTION
**Pull request by rafia17**

### CODE REVIEW

**Comment by EthanMarx** [comment]
Nice work. It looks like you don't have the pre-commit hooks installed. See [here](https://github.com/ML4GW/BBHNet/blob/main/CONTRIBUTING.md#pre-commit-hooks) in the contributing.md file for instructions on how to install those. Once installed, you should be able to run `pre-commit run --all-files` and then re-commit

**Comment by rafia17** [comment]
Hi there!

I have run all pre-commit checks and everything looks fine at my end. Not
sure what next to do in this pull request to have it merge. I think you
might need to approve or something.

Thanks
Rafia

**Review by EthanMarx — CHANGES_REQUESTED** [review]

File: projects/sandbox/vizapp/vizapp/plots/vt.py [hunk 1]
```diff
@@ -164,8 +167,19 @@ def calculate_vt(self, event):
...
+            tooltips=[
+                ("VT", "@vt"),
+                ("N_eff", "@n_eff"),
+                ("Uncertainty", "@uncertainty"),
+            ]
+        )
+        self.figure.add_tools(hover)
+
         self.figure.extra_y_ranges["n_eff"].start = 0.5 * n_eff.min()
         self.figure.extra_y_ranges["n_eff"].end = 2 * n_eff.max()
```
[the remark this answers is not in the document]
Comment (rafia17) [reply projects/sandbox/vizapp/vizapp/plots/vt.py]: Yes. I have removed those and pushed the changes to the branch.

[the remark this answers is not in the document]
Comment (rafia17) [reply projects/sandbox/vizapp/vizapp/plots/vt.py]: Yes, I have removed those and pushed changes to the branch

**Review by EthanMarx — APPROVED** [review]

**Pull request merged**